README.md

```diff
@@ -17,6 +17,7 @@ discount - The discount percentage to be applied to the converstion rate
 
 You can see a running example implementation here: 
 
+(EXAMPLE NO LONGER UP TO DATE - LEAVING HERE FOR REFERENCE)
```

Comment:
https://consensusreality.io/demo/app/pay.html uses the code from the master branch of the original repo, and as such it is up to date. Can we remove this line?